src/components/BookItem.jsx

```diff
@@ -48,14 +56,15 @@ const BookItem = ({ bookProp }) => {
           <button type="button">Comment</button>
           {' '}
           |
-          <button type="button">Remove</button>
+          <button type="button" onClick={handleRemove}>Remove</button>
           |
-          <button type="button" onClick={handleEdit}> Edit </button>
+          <button type="button" onClick={handleEdit}>Edit</button>
```

Comment:
- [ ] The remove button is not visible in your app hence I can't test if the remove function is working. Kindly fix this, please.
![image](https://github.com/emarkees/book-store/assets/94551484/6a0900fa-f368-4ee3-8838-663fc003587b)

Reply:
-  The button is now visible and I am able to delete books